examples/bri-3/src/bri/identity/bpiAccounts/subject.accounts.profile.ts

```diff
@@ -0,0 +1,18 @@
+import { AutomapperProfile, InjectMapper } from '@automapper/nestjs';
+import { createMap, Mapper } from '@automapper/core';
+import { Injectable } from '@nestjs/common';
+import { BpiAccountDto } from './api/dtos/response/bpiAccount.dto';
+import { BpiAccount } from './models/bpiAccount';
+
+@Injectable()
+export class AccountsProfile extends AutomapperProfile {
+  constructor(@InjectMapper() mapper: Mapper) {
+    super(mapper);
+  }
+
+  override get profile() {
+    return (mapper) => {
+      createMap(mapper, BpiAccount, BpiAccountDto);
```

Comment:
this can not be solved with @Automap annotation?

Reply:
createMap reads Automap annotation. so basically, we need both to work